packages/core/src/test/amazonqFeatureDev/controllers/chat/controller.test.ts

```diff
@@ -518,7 +522,13 @@ describe('Controller', () => {
                 )
                 const metricResult = getMetricResult(error)
                 assert.ok(
-                    sendMetricDataTelemetrySpy.calledWith(MetricDataOperationName.EndCodeGeneration, metricResult)
+                    sendMetricDataTelemetrySpy.calledWith(
+                        MetricDataOperationName.EndCodeGeneration,
+                        metricResult,
+
+                        // Stack trace should include the name of the test framework
+                        sinon.match((str) => typeof str === 'string' && str.includes('Mocha'))
```

Comment:
Can we assert the useful part of the log message is actually present? This does not guarantee that the stack trace is actually empty.